Challenge3/src/utils.ts

```diff
@@ -0,0 +1,87 @@
+import { Finding, FindingSeverity, FindingType, ethers, Alert, AlertsResponse, getAlerts } from "forta-agent";
+import { L2_DAI_TOKEN_ADDRESS, L2_TOKEN_ABI } from "./constants";
+import { Provider } from "@ethersproject/providers";
+
+export const l1Finding = (_l1OptEscrowBalance: string, _l1ArbEscrowBalance: string): Finding => {
```

Comment:
Should this function be updated so it only is used when one of the balances changed and also only show that balance's before and after amount?

Also, this function, along with `l2Finding` and `l2Finding`, could be moved to their own `findings.ts` file.